docassemble/MLHStatutoryWill/data/questions/MichiganStatutoryWill.yml

```diff
@@ -26,18 +26,18 @@ features:
   labels above fields: True
 ---
 sections:
-  - review_intro: "Introduction"
-  - do_you_qualify: "Do you qualify?"
-  - your_information: "Your Information"
-  - your_family: "Your Family"
-  - cash_gifts: "Cash Gifts"
-  - personal_and_household: "Personal and Household Items"
-  - residue: "Residue"
-  - personal_representatives: "Personal Representatives"
-  - guardians: "Guardians"
-  - conservators: "Conservators"
-  - bond: "Bond"
-  - done: "Done"
+  - signpost_intro: "Introduction"
```

Comment:
I like the leading keyword! We don't have a standard documented/chosen but we probably should agree on one.